### PR TITLE
CreateAccount and SingIn views changes

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/oTc4KUgl/714-ui-changes-in-signin-and-createaccount-views

I have no idea how to name that property `checkingIf: Bool = false` if you will have better ideas let me know 😃

### CODE REVIEW

File: AirCasting/AuthViews/CreateAccountView.swift [hunk 1]
```diff
@@ -37,63 +37,67 @@ struct CreateAccountView: View {
...
-                    titleLabel
-                    VStack(spacing: 20) {
-                        VStack(alignment: .leading, spacing: 5) {
-                            emailTextfield
-                                .keyboardType(.emailAddress)
-                            if !isEmailCorrect {
-                                errorMessage(text: AuthErrors.incorrectEmail.localizedDescription)
-                            }
+        ZStack(alignment: .bottomTrailing) {
+            Image("dashboard-background-thing")
```
Comment (jankrzempek) [inline AirCasting/AuthViews/CreateAccountView.swift:41]: This image goes uo with the kayboard, please have this in mind. IDK what is better here

File: AirCasting/AuthViews/SignInView.swift [hunk 2]
```diff
@@ -43,50 +43,54 @@ struct SignInView: View {
...
-                    titleLabel
-                    VStack(spacing: 20) {
-                        VStack(alignment: .leading, spacing: 5) {
-                            usernameTextfield
-                            if isUsernameBlank {
-                                errorMessage(text: AuthErrors.emptyTextfield.localizedDescription)
-                            }
+        ZStack(alignment: .bottomTrailing) {
+            Image("dashboard-background-thing")
+                .offset(x: 0, y: 40)
```
Comment (jankrzempek) [inline AirCasting/AuthViews/SignInView.swift:48]: same here with the image

File: AirCasting/Utils/Textfield.swift [hunk 3]
```diff
@@ -7,27 +7,27 @@
...
     .overlay(
         RoundedRectangle(cornerRadius: 8)
-            .stroke(Color.aircastingGray.opacity(0.1), lineWidth: 1)
+            .stroke(checkingIf ? .red : Color.aircastingGray.opacity(0.1), lineWidth: 1)
     )
     .background(RoundedRectangle(cornerRadius: 8).fill(Color.aircastingGray.opacity(0.05)))
 }
 
-func createSecuredTextfield(placeholder: String, binding: Binding<String>) -> some View {
+func createSecuredTextfield(placeholder: String, binding: Binding<String>, checkingIf: Bool = false) -> some View {
```
Comment (jankrzempek) [inline AirCasting/Utils/Textfield.swift:22]: maybe `shouldWarnUser`? if not please omit

Comment (SzymonGorowski) [reply AirCasting/Utils/Textfield.swift:22]: I love this idea, thanks!

File: AirCasting/Utils/Textfield.swift [hunk 4]
```diff
@@ -7,27 +7,27 @@
 
 import SwiftUI
 
-func createTextfield(placeholder: String, binding: Binding<String>) -> some View {
+func createTextfield(placeholder: String, binding: Binding<String>, shouldWarnUser: Bool = false) -> some View {
```
Comment (anna1901) [inline AirCasting/Utils/Textfield.swift:10]: I would call this argument `isInputValid`, cause I think that's the purpose of the red border (to show that input is not valid)

**Review by anna1901 — APPROVED** [review]
Nice! 👏🏻

File: AirCasting/Utils/Textfield.swift (hunk 4 above)
Comment (SzymonGorowski) [reply AirCasting/Utils/Textfield.swift:10]: Brilliant idea! Thanks!

**Pull request merged**